<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KNewInvestmentWizardDecl</class>
  <widget class="Q3Wizard" name="KNewInvestmentWizardDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>584</width>
        <height>385</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>New Investment wizard</string>
    </property>
    <widget class="QWidget" name="m_investmentTypePage">
      <attribute name="title">
        <string>Investment Type</string>
      </attribute>
      <layout class="QVBoxLayout">
        <item>
          <widget class="QLabel" name="m_introLabel">
            <property name="text">
              <string>This wizard allows you to create a new investment.</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item>
          <spacer name="spacer8">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>10</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="QLabel" name="textLabel1">
            <property name="text">
              <string>The first step in this process requires to select the type of investment. The following steps collect more details about the investment from you.</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item>
          <spacer name="spacer3">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>30</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
          </spacer>
        </item>
        <item>
          <layout class="QHBoxLayout">
            <item>
              <widget class="QLabel" name="textLabel2">
                <property name="text">
                  <string>Type of investment</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item>
              <widget class="KComboBox" name="m_securityType">
                <item>
                  <property name="text">
                    <string>Stock</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>Mutual Fund</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>Bond</string>
                  </property>
                </item>
              </widget>
            </item>
            <item>
              <spacer name="spacer1">
                <property name="sizeHint">
                  <size>
                    <width>61</width>
                    <height>21</height>
                  </size>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </item>
        <item>
          <spacer name="spacer2">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>60</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
          </spacer>
        </item>
      </layout>
    </widget>
    <widget class="QWidget" name="m_investmentDetailsPage">
      <attribute name="title">
        <string>Investment details</string>
      </attribute>
      <layout class="QVBoxLayout">
        <item>
          <widget class="QLabel" name="textLabel3_2">
            <property name="text">
              <string>Enter the details below and click &lt;b&gt;Next&lt;/b&gt; to continue entering the online update details.</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item>
          <spacer name="spacer6">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>16</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
          </spacer>
        </item>
        <item>
          <layout class="QGridLayout">
            <item row="1" column="1">
              <widget class="kMyMoneyLineEdit" name="m_investmentName"/>
            </item>
            <item row="2" column="1">
              <layout class="QHBoxLayout">
                <item>
                  <widget class="QLabel" name="textLabel1_5">
                    <property name="sizePolicy">
                      <sizepolicy>
                        <hsizetype>1</hsizetype>
                        <vsizetype>5</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>1 /</string>
                    </property>
                    <property name="alignment">
                      <set>Qt::AlignVCenter|Qt::AlignRight</set>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="kMyMoneyEdit" name="m_fraction"/>
                </item>
              </layout>
            </item>
            <item row="6" column="0">
              <widget class="QLabel" name="textLabel2_5">
                <property name="text">
                  <string>Price entry</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="textLabel2_2_2">
                <property name="text">
                  <string>Fraction</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="5" column="0">
              <widget class="QLabel" name="textLabel1_4">
                <property name="text">
                  <string>Trading Currency</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="textLabel3">
                <property name="text">
                  <string>Full name</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="6" column="1">
              <widget class="KMyMoneyGeneralCombo" name="m_priceMode"/>
            </item>
            <item row="3" column="1">
              <widget class="KComboBox" name="m_tradingMarket">
                <property name="editable">
                  <bool>true</bool>
                </property>
                <item>
                  <property name="text">
                    <string>AMEX</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>EUREX</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>FUND</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>NASDAQ</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>NYSE</string>
                  </property>
                </item>
                <item>
                  <property name="text">
                    <string>XETRA</string>
                  </property>
                </item>
              </widget>
            </item>
            <item row="0" column="0">
              <widget class="QLabel" name="textLabel2_2">
                <property name="text">
                  <string>Trading symbol</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="0">
              <widget class="QLabel" name="textLabel1_2">
                <property name="text">
                  <string>Identification</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="5" column="1">
              <widget class="KMyMoneySecuritySelector" name="m_tradingCurrencyEdit">
                <property name="displayType">
                  <enum>KMyMoneySecuritySelector::TypeCurrencies</enum>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="kMyMoneyLineEdit" name="m_investmentSymbol">
                <property name="toolTip" stdset="0">
                  <string>Enter the ticker symbol (e.g. RHAT).</string>
                </property>
              </widget>
            </item>
            <item row="4" column="1">
              <widget class="kMyMoneyLineEdit" name="m_investmentIdentification">
                <property name="toolTip" stdset="0">
                  <string>Enter the CUSIP/ISIN/WKN identification number here</string>
                </property>
              </widget>
            </item>
            <item row="3" column="0">
              <widget class="QLabel" name="textLabel1_3">
                <property name="text">
                  <string>Trading market</string>
                </property>
                <property name="wordWrap">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </item>
        <item>
          <spacer name="spacer5">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>16</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
          </spacer>
        </item>
      </layout>
    </widget>
    <widget class="QWidget" name="m_onlineUpdatePage">
      <attribute name="title">
        <string>Online Update</string>
      </attribute>
      <layout class="QVBoxLayout">
        <item>
          <widget class="QLabel" name="textLabel6">
            <property name="text">
              <string>Select an online source and click &lt;b&gt;Finish&lt;/b&gt; to store the investment data. If you don't want to use online updates, just leave the data as is.</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item>
          <layout class="QHBoxLayout">
            <property name="margin">
              <number>0</number>
            </property>
            <property name="spacing">
              <number>0</number>
            </property>
            <item>
              <layout class="QVBoxLayout">
                <item>
                  <layout class="QHBoxLayout">
                    <item>
                      <widget class="QLabel" name="textLabel2_4">
                        <property name="text">
                          <string>Use Finance::Quote</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="m_useFinanceQuote">
                        <property name="text">
                          <string/>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <layout class="QHBoxLayout">
                    <item>
                      <widget class="QLabel" name="textLabel2_3">
                        <property name="text">
                          <string>Online Source</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="KComboBox" name="m_onlineSourceCombo">
                        <property name="minimumSize">
                          <size>
                            <width>180</width>
                            <height>0</height>
                          </size>
                        </property>
                        <item>
                          <property name="text">
                            <string/>
                          </property>
                        </item>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <layout class="QHBoxLayout">
                    <item>
                      <widget class="QLabel" name="textLabel1_6">
                        <property name="text">
                          <string>Factor</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="kMyMoneyEdit" name="m_onlineFactor"/>
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
            <item>
              <spacer name="spacer29">
                <property name="sizeHint">
                  <size>
                    <width>371</width>
                    <height>20</height>
                  </size>
                </property>
                <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                </property>
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
              </spacer>
            </item>
          </layout>
        </item>
        <item>
          <spacer name="spacer30">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>181</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
          </spacer>
        </item>
      </layout>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>kMyMoneyLineEdit</class>
      <extends>QWidget</extends>
      <header location="local">../widgets/kmymoneylineedit.h</header>
      <sizehint>
        <width>90</width>
        <height>25</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">kcombobox.h</header>
    </customwidget>
  <customwidget>
   <class>KMyMoneyGeneralCombo</class>
   <extends>QWidget</extends>
   <header>kmymoneymvccombo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KMyMoneySecuritySelector</class>
   <extends>QWidget</extends>
   <header>kmymoneycurrencyselector.h</header>
   <container>1</container>
  </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>m_securityType</tabstop>
    <tabstop>m_investmentSymbol</tabstop>
    <tabstop>m_investmentName</tabstop>
    <tabstop>m_fraction</tabstop>
    <tabstop>m_tradingMarket</tabstop>
    <tabstop>m_investmentIdentification</tabstop>
    <tabstop>m_tradingCurrencyEdit</tabstop>
    <tabstop>m_priceMode</tabstop>
    <tabstop>m_useFinanceQuote</tabstop>
    <tabstop>m_onlineSourceCombo</tabstop>
    <tabstop>m_onlineFactor</tabstop>
  </tabstops>
  <images>
    <image name="image0">
      <data format="PNG" length="1110">89504e470d0a1a0a0000000d4948445200000016000000160806000000c4b46c3b0000041d49444154388d8d95c18b1c4514c67fbb29e5b54ea05b26da1d36b08d09a6c508138cb8c13d647141163de49043bca9f1628eb9294810f24f88c15b728b07617358888785d980427290b4a8d00b19a8821da8228cf42316ac87eed9247ac95c9aeaeef9eaabdffb5ebf85f178ccfcf7f5b79fec07a7280a08a008f25ceb7c29e5da573716e65a0be3f198ef6f5ededff9b9e1b34faf90882092a1ea21428b9220ffbf46054055b19386ba6970ae6665b5e0d2c5eb0b066067bbe1f21757c88f5504eff0c192189081303a5582b16854fccce2fe041f00cdd0a8103b874556405476b62d972ec2c2b90ff3fd8df5cb946549d80b34aea15ccaa94625450e2d0e558b460b7371a734b5e027251a155545678ab62d7eea90b4c504078908c107acb354cb251b1f8f08b1c18631aabbb4d18211404804f22545863561d2f060fb34aa8a18d0086204e72c46a32292e1a796f278c1da7a859d8da9a737506db04d0a24103d8949480b214d3bbee952a05c1d536f9d2684160c744f048311543d459eb1b65ee1754cedae13a64abd5d6127053c9d0281f22d4f75c6a2118a5c09272d7ab740638bf4ef2dd2174032d0d8d04c6fe19c32bedd8bc65e342aa3f785e2784bfd6bc1bd9db24b05505616a4edd67d5a8c1801a03a9962754c504bf34b8986acfb9be91cac7d54315a15541b36b134f74bd2dc53940115a5381e087bc25c6f71be831a8baa234cc1ee3eeb74ed83116beb2b8849d108a3730e8ce27e2bfb82419a7b407ac7c222fd0e6280a8078ce64e47ef54ac9d5be9aaae0162e0de568a44a18d3df90845d919c4f01463406340a44f40efb43a99b371618488e2670d1a2d7ee6f1d3be39342104e937ed85fb86599c33e9ee0be9a004946259d8b858909880d59aa00daa0d3280b5f39e34074c4b3a5434029a76a9e9f5cc0163b5c80024158a1cfc4cb1d3061d5834066cb0a403c008599a70f6bc27b8d03106ea3ae91987278c5b143b015547224ab5e24185cd9b81665273e7b6e7cecd82ddbf8098002de94049737bc09828cf3036f31c3735c8b0e98eb704672f348c7f7c93cd1fb2ae905119ff54e0dff3542b0e9dd983dcea2cc5eea6cf303e74ec687af5c41b6fa38f5e203c7ec8e1238ef85831a214afef21060ea781ec68247b65c6916353cc8b73d1ae79354606af2a930709aa118da1636c274dd7e3774b884a79aa6b5711a538d33cf9c01c1c9b83bc6a1f55c91ccb23b05b1deb43a7decdaffefd2862f61366aa8449c69e33642f012f2bb208fa98eeda1f3346f076c8eeef19475e9be115423364f78f84105a8643d34d90cfbfdcd8cf872589a45dc7f54cf3134a36f4e495c23f9da8db15bccb708d4014f2ca512cb5d4f7c13e84300d5cff6eb39b202bab053bdb9674f0648609427d0f8402ddeac30f4fcd3ced3efc3b422d406c09b340bedc1571e1bfc3d44dc281634c5fe5e758e7395cfbe6d6c130fd17d92372d277c159b70000000049454e44ae426082</data>
    </image>
  </images>
</ui>
